pkg/config/render_config.go

```diff
@@ -82,6 +83,7 @@ func mkContext(buildType string) context {
 		Compliance:        true,
 		SNMP:              true,
 		PrometheusScrape:  true,
+		OpenTelemetry:     true,
```

Comment:
I don't understand why we need this

Reply:
I have no idea. All sections have it. Perhaps we should've conditioned it with the `otlp` build tag, but since that's apparently always there, it's fine to remove it.